### ISSUE TITLE
"Configuration as Code Plugin" cannot handle the collectNodeStatus attribute

### ISSUE THREAD

**Issue by con-vissenberg** [jenkinsci/prometheus-plugin#525]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
- Jenkins 2.378.3
- Configuration as Code Plugin 1625.v27444588cc3d
- Prometheus-plugin  2.2.2 
  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

Ubuntu 20.04

### Reproduction steps

Use export function of JCasC.

### Expected Results

```
# Removed all unrelated data
unclassified:
  prometheusConfiguration:
    appendParamLabel: false
    appendStatusLabel: false
    collectDiskUsage: true
    collectingMetricsPeriodInSeconds: 120
    collectNodeStatus: false
    countAbortedBuilds: true
    countFailedBuilds: true
    countNotBuiltBuilds: true
    countSuccessfulBuilds: true
    countUnstableBuilds: true
    defaultNamespace: "default"
    fetchTestResults: true
    jobAttributeName: "jenkins_job"
    path: "prometheus"
    perBuildMetrics: false
    processingDisabledBuilds: false
    useAuthenticatedEndpoint: false

```

### Actual Results

```
# Removed all unrelated data
unclassified:
  prometheusConfiguration:
    appendParamLabel: false
    appendStatusLabel: false
    collectDiskUsage: true
    collectingMetricsPeriodInSeconds: 120
    countAbortedBuilds: true
    countFailedBuilds: true
    countNotBuiltBuilds: true
    countSuccessfulBuilds: true
    countUnstableBuilds: true
    defaultNamespace: "default"
    fetchTestResults: true
    jobAttributeName: "jenkins_job"
    path: "prometheus"
    perBuildMetrics: false
    processingDisabledBuilds: false
    useAuthenticatedEndpoint: false

```

### Anything else?

When I add `collectDiskUsage: false` to the YAML file and have JCasC process that file, JCasC crashes with a "unknown attribute" error.

**Comment by Waschndolos** [comment]
Thanks for the report

**Comment by Waschndolos** [comment]
Good news: I found a fix for that. Bad news: I broke some tests and run out of time for today :) I'll continue either tomorrow or at the weekend. A fix should be available on Monday then.

**Comment by con-vissenberg** [comment]
Thank you. Nice to get such quick feedback!

**Comment by Waschndolos** [comment]
Fixed with [Release 2.2.3](https://github.com/jenkinsci/prometheus-plugin/releases/tag/prometheus-2.2.2)

**Comment by con-vissenberg** [comment]
Thank you. I can confirm that it works now.
